src/Cli.Tests/EndToEndTests.cs

```diff
@@ -326,6 +326,36 @@ public void TestUpdateDepthLimitInGraphQLRuntimeSettings(string depthLimit, bool
         Assert.AreEqual(isSuccess, isError == 0);
     }
 
+    /// <summary>
+    /// This test checks behavior of executing `dab configure --runtime.graphql.{key} value
+    /// </summary>
+    [DataTestMethod]
+    [DataRow("/updatedPath", true, DisplayName = "Success in updating graphQL path values.")]
+    [DataRow("/updated-Path", true, DisplayName = "Success in updating graphQL path values.")]
+    [DataRow("/updated_Path", true, DisplayName = "Success in updating graphQL path values.")]
```

Comment:
Can you slightly change each of the DisplayNames to something like "Success in updating graphQL 'updatedPath'/'updated-Path'/'updated_Path' path values." so that you can differentiate them when looking for the tests

Reply:
updated